src/lib/commands.ts

```diff
@@ -167,4 +170,61 @@ export const registerCommands = (context: vscode.ExtensionContext) => {
       }
     )
   );
+
+  context.subscriptions.push(
+    vscode.commands.registerCommand(COMMANDS.INSTALL_HUBSPOT_CLI, () => {
+      const terminal = vscode.window.createTerminal(TERMINAL_IDS.CLI_INSTALL);
+
+      terminal.hide();
+      vscode.window.onDidCloseTerminal(async (closedTerminal) => {
+        if (closedTerminal.name === TERMINAL_IDS.CLI_INSTALL) {
+          const retVal = await vscode.commands.executeCommand(
+            COMMANDS.CHECK_HUBSPOT_CLI_INSTALL
+          );
+          console.log('retVal: ', retVal);
+          if (retVal) {
+            vscode.window.showInformationMessage('HubSpot CLI installed.');
+          } else {
+            vscode.window.showInformationMessage(
+              'Error Installing HubSpot CLI.'
+            );
+          }
+        }
+      });
+      terminal.sendText("echo 'Installing the HubSpot CLI.'");
+      terminal.sendText('npm i -g @hubspot/cli@latest');
```

Comment:
Should we allow them to specify npm / yarn /  other ?

Reply:
Do you think that multiple buttons would be the way to go, or have a quickpick open up when clicking the button 🤔 

Multiple buttons
<img width="705" alt="image" src="https://user-images.githubusercontent.com/6472448/204849925-16d00fb3-7604-414b-8978-265c214537a2.png">

Quickpick
<img width="1920" alt="image" src="https://user-images.githubusercontent.com/6472448/204851141-80edfff0-b45e-4b3f-b98b-3fffb65bd90a.png">

Reply:
Yeah I did see that Stripe was sending folks to the docs instead of doing the installation. I was going by the description of the issue -> https://git.hubteam.com/HubSpot/cms-developer-tooling/issues/54 and discussion with @hs-cboamah .

There is a link above the button that leads to https://developers.hubspot.com/docs/cms/developer-reference/local-development-cli. Here is the text with the button under it...

```
In order to use additional features that are not currently available within the HubSpot VSCode extension, you can install the [HubSpot CLI](https://developers.hubspot.com/docs/cms/developer-reference/local-development-cli). \n[Install CLI](command:hubspot.hs.install)
```

Should we change the text?

Reply:
I think keeping the button is totally fine -- let's just adjust the text around the installation button.

"In order to use additional features that are not currently available within the HubSpot VS Code extension, you can install the HubSpot CLI (via npm) using the button below. To see other available installation options, check out the documentation."

We'll have to follow up with additions/changes/updates to the docs in order to support this. Probably a CMS Developer Doc page dedicated to the extension would help. Otherwise, in the interim, we can create some quick documentation that lives in the readme of the repo for CLI (or extension) in order to send devs to.

Reply:
Updated!

What kind of info do you see this doc containing?